src/elements/Svg.tsx

```diff
@@ -45,6 +45,40 @@ export interface SvgProps extends GProps, ViewProps, HitSlop {
   title?: string;
 }
 
+export type DataUrlOptions = JpegOptions | PngOptions;
+
+interface JpegOptions {
+  format: 'jpeg';
+  quality?: number;
+  width: NumberProp;
+  height: NumberProp;
+}
+
+interface PngOptions {
+  format: 'png';
+  width: NumberProp;
+  height: NumberProp;
+}
+
+function checkOptions(options?: DataUrlOptions) {
+  if (options && options?.format === 'jpeg') {
+    if (!validateJpegQualityParameter(options)) {
+      throw new Error('toDataURL: Invalid quality parameter for JPEG format.');
+    }
+  }
+}
+
+function validateJpegQualityParameter(options: JpegOptions): boolean {
+  if (options.quality && (options.quality < 0 || options.quality > 1)) {
+    return false;
+  }
+  // Android requires quality to be a number between 0 and 100
+  if (Platform.OS === 'android' && options.quality) {
```

Comment:
Redundant check. If `options.quality` is a falsy value, the function will `return false` in the previous condition

Reply:
You're right, thank you ;)